content/pages/2023/sponsors.md

```diff
@@ -10,4 +10,5 @@ slug: sponsoring_2023
 ### Links
 
 - [Become a Sponsor](sponsoring.html)
+- [Current Sponsors](current_sponsors.html)
```

Comment:
This referring the file above, right? then maybe `current_sponsors.html`? https://github.com/euroscipy/euroscipy.github.io/blob/feature/sponsoring/content/pages/2023/current_sponsors.md